consensus/e2e_tests/state_sync_test.go

```diff
@@ -134,28 +132,53 @@ func TestStateSyncServerGetBlock(t *testing.T) {
 	require.NotEmpty(t, getBlockRes)
 
 	require.Equal(t, uint64(1), getBlockRes.Block.GetBlockHeader().Height)
+}
+
+func TestStateSync_ServerGetBlock_FailingTest(t *testing.T) {
+	// Test preparation
+	clockMock := clock.NewMock()
+	timeReminder(t, clockMock, time.Second)
+
+	// Test configs
+	runtimeMgrs := GenerateNodeRuntimeMgrs(t, numValidators, clockMock)
+	buses := GenerateBuses(t, runtimeMgrs)
+
+	// Create & start test pocket nodes
+	eventsChannel := make(modules.EventsChannel, 100)
+	pocketNodes := CreateTestConsensusPocketNodes(t, buses, eventsChannel)
+	StartAllTestPocketNodes(t, pocketNodes)
+
+	testHeight := uint64(5)
+
+	serverNode := pocketNodes[1]
+	serverNodeConsensusModImpl := GetConsensusModImpl(serverNode)
+	serverNodeConsensusModImpl.MethodByName("SetHeight").Call([]reflect.Value{reflect.ValueOf(testHeight)})
```

Comment:
Rather than using reflection, can you not just change `GetConsensusModImpl` to `GetConsensusPacacmekerModImpl`?

Reply:
Pacemaker in consensus module is not exported, and `ConsensusPacemaker` interface is implemented for Consensus Module, so we need to access pacemaker related functions, and also state sync related functions via the consensus module.